calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -318,15 +318,34 @@ kubectl annotate svc <service-name> -n <service-namespace> projectcalico.org/l7-
 
 ### Disable WAF feature from your cluster
```

Comment:
### Disable WAF feature from your cluster

To safely disable WAF, determine how to handle ApplicationLayer features and follow the steps:

- **Disable all ApplicationLayer features**

   You must delete the ApplcationLayer CR `tigera-secure`, for example:

    `kubectl delete applicationlayer tigera-secure`


- **Keep some ApplicationLayer features enabled**

To disable WAF, but keep some ApplicationLayer features enabled, you must update the [ApplicationLayer](../reference/installation/api#operator.tigera.io/v1.ApplicationLayer). Note that the [ApplicationLayer Specification](../reference/installation/api#operator.tigera.io/v1.ApplicationLayerSpec) can specify configuration for both [Application Logging](../reference/installation/api#operator.tigera.io/v1.LogCollectionSpec) and [Application Layer Policy](../reference/installation/api#operator.tigera.io/v1.ApplicationLayerPolicyStatusType). 

For the ApplicationLayer custom resource to be valid, at least one of these features must be enabled, for example:

**Valid YAML**

WAF is enabled, and is ALP disabled.

```yaml
apiVersion: operator.tigera.io/v1
kind: ApplicationLayer
metadata:
  name: tigera-secure
spec:
  webApplicationFirewall: Disabled
  applicationLayerPolicy: Enabled
```

**Invalid YAML**

WAF and ALP are disabled, and log collection is omitted (and the default is disabled). 

```yaml
apiVersion: operator.tigera.io/v1
kind: ApplicationLayer
metadata:
  name: tigera-secure
spec:
  webApplicationFirewall: Disabled
  applicationLayerPolicy: Disabled
```